addons/scratchstats/addon.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Profile statistics",
+  "description": "Shows statistics from ScratchDB on profile pages.",
```

Comment:
Similarly to other addons that use ScratchDB (or that fetch any other external resource), we should specify this uses ScratchDB and ScratchStats.com (since the img icon is fetched from it)

Reply:
I'd rather include the icon in the addon directory then - the statistics themselves aren't from ScratchStats.com and I don't want the description to be misleading.

Reply:
Sure include the icon